packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -643,6 +653,11 @@ export class ExistingQueryEditorStore extends QueryEditorStore {
     this.queryBuilderState.initialize(
       await this.graphManagerState.graphManager.pureCodeToLambda(query.content),
     );
+    await Promise.all([
```

Comment:
See my comment in `QueryBuilderChangeDetectionState`, we should only need to do:
`await queryBuilderState.changeDetectionState.start();` here